app/views/developers/_form.html.erb

```diff
@@ -39,6 +39,13 @@
               <p class="mt-2 text-sm text-gray-500"><%= t(".profile.bio_help_text") %></p>
             </div>
 
+            <div class="sm:col-span-6">
+              <%= form.label :time_zone, class: "block text-sm font-medium text-gray-700" %>
+              <div class="mt-1">
+                <%= form.time_zone_select(:time_zone, ActiveSupport::TimeZone.us_zones, include_blank: true) %>
+              </div>
+            </div>
```

Comment:
<img width="465" alt="image" src="https://user-images.githubusercontent.com/2092156/145256596-db811769-d0c6-4e2a-ae57-a1e95f424d43.png">

Can you style this? The screenshot is from desktop Safari.

Tailwind UI has a [Simple native select menu](https://tailwindui.com/components/application-ui/forms/select-menus#component-79592f355ca6f6e3b081e0158109981c) that doesn't require JavaScript. If you don't have access let me know and I can add the classes myself.

Reply:
This has been updated as well